ruby/ql/lib/codeql/ruby/ast/Erb.qll

```diff
@@ -252,14 +252,32 @@ class ErbGraphqlDirective extends ErbDirective {
 class ErbOutputDirective extends ErbDirective {
   private Erb::OutputDirective g;
 
-  ErbOutputDirective() { this = TOutputDirective(g) }
+  ErbOutputDirective() { this = TOutputDirective(g) or this = TRawOutputDirective(g) }
 
   override ErbCode getToken() { toGenerated(result) = g.getChild() }
 
+  /**
+   * Holds if this is a raw Erb output directive.
+   * ```erb
+   * <%== foo %>
+   * ```
+   */
+  predicate isRaw() { this = TRawOutputDirective(g) }
```

Comment:
Have you tried the following? This way you don't need any of the changes related to adding `TRawOutputDirective`

```suggestion
  predicate isRaw() {
    exists(Erb::Token t | t.getParentIndex() = 0 and t.getParent() = g and t.getValue() = "<%==")
  }
```

Reply:
That's a nice simplification!